src/timeseriesinsights/README.md

```diff
@@ -1,95 +1,129 @@
-Microsoft Azure CLI 'timeseriesinsights' Extension
-==========================================
+# Azure CLI timeseriesinsights Extension #
+This is the extension for timeseriesinsights
 
-This package is for the 'timeseriesinsights' extension:
-
-```sh
-az timeseriesinsights
+### How to use ###
+Install this extension using the below CLI command
 ```
-
-## Install
-
-```sh
 az extension add --name timeseriesinsights
 ```
 
-## Uninstall
-
-```sh
-az extension remove --name timeseriesinsights
+### Included Features ###
+#### timeseriesinsights environment ####
+##### Gen1 Create #####
 ```
-
-## Examples
-
-All commands are for Bash. For PowerShell, please set the variables like 
-
-```powershell
-$rg='rg1'
+az timeseriesinsights environment gen1 create --name "env1" --location westus --data-retention-time "P31D" \
+    --partition-key-properties name="DeviceId1" type="String" --sku name="S1" capacity=1 --resource-group "rg1"
 ```
 
-### Create a resource group for the environments
-
-```sh
-rg={resource_group_name}
-az group create --name $rg --location westus
+##### Gen1 Update #####
 ```
-
-### Create a standard environment
-
-```sh
-env={standard_environment_name}
-az timeseriesinsights environment standard create -g $rg --name $env --location westus --sku-name S1 --sku-capacity 1 --data-retention-time 31 --partition-key-properties DeviceId1 --storage-limit-exceeded-behavior PauseIngress
+az timeseriesinsights environment gen1 update --name "env1" --sku name="S1" capacity=2 --resource-group "rg1" \
```

Comment:
Whitespace is used to separate objects in a list. If I remember correctly, we have agreed to use comma `,` to separate properties of one object.

```suggestion
az timeseriesinsights environment gen1 update --name "env1" --sku name="S1",capacity=2 --resource-group "rg1" \
```